File: SummaMoveAdmin/SummaMoveAdmin/Models/Oefening.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SummaMoveAdmin.Models
{
   public class Oefening
    {
        public int ID { get; set; }
        public string Naam { get; set; }
        public string Beschrijving { get; set; }
        public byte[] Foto { get; set; }
        
    }
}

File: project5api/app/Models/Oefening.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Oefening extends Model
{
    use HasFactory;

    protected $table = "oefening";
    public $timestaps = false;
    protected $fillable = [
        'naam','beschrijving', 'foto',
    ];
}
